src/source-pcap-file.c

```diff
@@ -44,6 +44,7 @@
 #include "runmode-unix-socket.h"
```

Comment:
think we can split this file? I think it's getting a bit hard to follow all the logic. Perhaps a src/source-pcap-file-dir or src/source-pcap-dir?

Reply:
I'm looking at leaving source-pcap-file in place focusing on the receive/decode thread functionality, and introducing pcap-file-helper and pcap-directory-helper with the implementation related to that functionality.

Reply:
yeah makes sense